.env.example

```diff
@@ -1,66 +1,82 @@
-SERVER_PORT=4000
+##############################################################
+#                      COMPANY DETAILS                       #
+##############################################################
+
+COMPANY_NAME=Example Company
+COMPANY_EMAIL=aaa@example.com
+DEFAULT_PASSWORD=Password123&
```

Comment:
Having a pre-populated `.env` would have enabled me to run the project straight away or at least to prevent me having to investigate how the code works just to get it running. I think having a `.env.example` or `.env.local` or something in source control would be hugely valuable for quickly getting up and running locally, assuming others are expected to run this project. I'd suggest having these values somewhere, but I think the best place for them is likely in version control rather than some external document.

Reply:
Yeah, in fact, we need those setting up to at least take a look at the ReadMe and have an idea of what they're setting up, we already have the .env.example in source control to quickly get started, but we cant have a default passoword for use there, the first comment is about the password values, not about the .env.example file as a whole. Pretty much asking that the following values be left blank in the .env.example files; COMPANY_NAME, COMPANY_EMAIL, DEFAULT_PASSWORD, ELASTICSEARCH_PASSWORD

Reply:
That makes sense to me. I'll update the `.env.example` and `README.md`.